code/modules/crafting/smeltery.dm

```diff
@@ -0,0 +1,46 @@
+/*Crafting recipes for stuff made out of the blacksmtihing tools*/
+
+/datum/crafting_recipe/broadsword
+	name = "Smith Broadsword"
```

Comment:
bit of inconsistency here, not sure if its intentional but some items are 'smithed' while some are 'smith', like 'smith broadsword' sounds weird; should it be smithed? Same with the armor and helmet?